modules/op-starting-pipelines.adoc

```diff
@@ -0,0 +1,48 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc
+
+[id="op-starting-pipelines_{context}"]
+= Starting Pipelines
+
+After you create a Pipeline, you need to start it for the Pipeline to run and execute its Tasks. You can start a Pipeline Run from the *Pipelines* view, *Pipeline Details* page, or the *Topology* view.
```

Comment:
s/After you create a Pipeline,you need to start it for the Pipeline to run and execute its Tasks./After you create a Pipeline, you need to start it in order to execute the included Tasks in the defined sequence ?